source/strings.tex

```diff
@@ -275,10 +275,10 @@
 \pnum
 \requires
 \tcode{state_type}
-shall satisfy the requirements of
+shall satisfy the
 \tcode{CopyAssignable} (\tref{copyassignable}),
```

Comment:
This is one of the few cases where the placement of the reference is acceptable, since you are listing multiple requirements. In all cases where we only say *one* requirement, I would much prefer the reference *after* the word "requirements". Otherwise it's really disrutpive: "shall satisfy the Foo (Table n) requirements." vs "shall satisfy the Foo requirements (Table n)."

We even have a lot of precedent for that placement in the many other cases where we already say " shall satisfy the Foo requirements". So it wouldn't just read better, but also be more consistent.

Reply:
Fixed.